### PR TITLE
feat:Setup of Express Complete(39521)

### PR DESCRIPTION
**Pull request by Rajshrishrimalshah**
please review my assignment.

### CODE REVIEW

File: extra/utils/helpers.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+// export const validateEmail = (users) => {
+//   let myEmail = /@successive.tech\s*$/;
+
+//   return users.match(myEmail);
+
+
+//   };
+export function validateEmail(users){
+  console.log(users)
+  let myEmail = /@successive.tech\s*$/;
```
Comment (SunilSuccessive) [inline extra/utils/helpers.js:10]: this regex will return true for "raj.shri@successive.tech" and also true for "@successive.tech" but for this i should return false

File: src/config/configuration.ts [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import { IConfig } from './IConfig';
+import * as dotenv from 'dotenv';
+dotenv.config();
+
+const configuration = {
```
Comment (SunilSuccessive) [inline src/config/configuration.ts:5]: Define type

File: src/Server.ts [hunk 3]
```diff
@@ -0,0 +1,29 @@
+import { IConfig } from "./config/IConfig";
```
Comment (SunilSuccessive) [inline src/Server.ts:1]: Should be import from config folder only

File: src/index.ts [hunk 4]
```diff
@@ -0,0 +1,5 @@
+import { default as configuration } from "./config/configuration";
```
Comment (SunilSuccessive) [inline src/index.ts:1]: Should be import from config folder only

File: src/Server.ts [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+import * as express from "express";
+
+class Server {
+  private port;
+  app = express();
+
+  constructor(config: IConfig) {
+    this.port = process.env.PORT;
```
Comment (SunilSuccessive) [inline src/Server.ts:9]: Don't use this because u can get env data from config file

File: src/index.ts [hunk 6]
```diff
@@ -0,0 +1,5 @@
...
+import Server from "./Server";
+
+const server = new Server(configuration);
```
Comment (SunilSuccessive) [inline src/index.ts:4]: Define types of server const

File: src/Server.ts [hunk 7]
```diff
@@ -0,0 +1,29 @@
+import { IConfig } from "./config/IConfig";
+import * as express from "express";
+
+class Server {
+  private port;
+  app = express();
```
Comment (SunilSuccessive) [inline src/Server.ts:6]: app should be initialize inside constructor

**Review by SunilSuccessive — CHANGES_REQUESTED** [review]

File: src/Server.ts (hunk 7 above)
Comment (Rajshrishrimalshah) [reply src/Server.ts:6]: done

File: src/config/configuration.ts (hunk 2 above)
Comment (Rajshrishrimalshah) [reply src/config/configuration.ts:5]: done.

File: src/Server.ts (hunk 3 above)
Comment (Rajshrishrimalshah) [reply src/Server.ts:1]: done.

File: src/index.ts (hunk 4 above)
Comment (Rajshrishrimalshah) [reply src/index.ts:1]: done.

File: src/Server.ts (hunk 5 above)
Comment (Rajshrishrimalshah) [reply src/Server.ts:9]: done.

File: extra/utils/helpers.js (hunk 1 above)
Comment (Rajshrishrimalshah) [reply extra/utils/helpers.js:10]: done.

File: src/index.ts (hunk 6 above)
Comment (Rajshrishrimalshah) [reply src/index.ts:4]: done.

**Pull request merged**